app/bundles/LeadBundle/Entity/Lead.php

```diff
@@ -199,6 +200,9 @@ class Lead extends FormEntity implements CustomFieldEntityInterface
      */
     private $color;
 
+    /** @var LeadManipulator */
```

Comment:
Can this annotation use the new lines as all of the annotations above and bellow?

Reply:
ok, it's just so long..